merlin-server/src/test/java/gov/nasa/jpl/aerie/merlin/server/models/MissionModelTest.java

```diff
@@ -46,7 +45,7 @@ public void shouldGetActivityTypeList() {
                     new Parameter("y", ValueSchema.STRING),
                     new Parameter("vecs", ValueSchema.ofSeries(ValueSchema.ofSeries(ValueSchema.REAL)))),
                 List.of(),
-                new UnitValueMapper().getValueSchema()
+                ValueSchema.ofStruct(Map.of())
```

Comment:
I know you already mentioned that this is very brittle, @Twisol, but I'm highlighting this for other reviewers, since your description doesn't mention that part of the brittleness comes from inlining certain methods from `contrib` classes.

Reply:
The brittleness I was referring to is from the delicate nature of the classloader situation, where we need to avoid having CSPICE on the classpath at all costs.

I'd claim that this particular line is also not brittle, because we're testing specifically using the `foo-missionmodel` model, so if `foo-missionmodel` changes its representation, these tests will instantly break and we will update the tests, same as any coupling between unit tests and the unit they test.

I'll add information about the brittleness of the scheduler's `Duration` usage in the "Future work" section in a moment.

Reply:
Understood. My concern came from inlining also occurring`SimulationFacade` and `GraphQLMerlinService` with the DurationValueMapper, which are used are used outside of testing (which you have now added to the `Future Work` section.

Reply:
Right, agreed. Since those other uses (unlike this one) all seem to be in the scheduler, I think it's fair to say that the future work I outlined will address that. Thanks for the detailed check :+1: